src/org/thoughtcrime/securesms/components/ScaleStableImageView.java

```diff
@@ -85,88 +85,33 @@ protected void onSizeChanged(int width, int height, int oldWidth, int oldHeight)
             Log.i(TAG, "orientation was: " + orientation);
             return; // something fishy happened.
         }
-
         if (!(defaultDrawable instanceof BitmapDrawable)) {
             return; // need Bitmap for scaling and cropping.
         }
 
-        measureViewSize(width, height, oldWidth, oldHeight, portrait);
+        Log.i(TAG, "view size. w: " + width + " h: " + height + " p: " + portrait);
```

Comment:
maybe comment these out for now (instead of removing them completely)?

Reply:
I removed all the debugging comments in the cleanup commit.